### ISSUE TITLE
`Uncaught (in promise) Error: Schema not found` on new installation

### ISSUE THREAD

**Issue by TomAugspurger** [dask/dask-labextension#191]
**What happened**:

I create a new environment

```
conda create -n dask-labextension-test -c conda-forge python=3.9 ipython jupyter notebook jupyterlab dask-labextension
```

and start jupyterlab, and I see an error in the browser console

```
Uncaught (in promise) Error: Schema not found: /home/taugspurger/miniconda3/envs/dask-labextension-test/share/jupyter/lab/schemas/dask-labextension/plugin.json
    at Function.create (static/lab/jlab_core.4cb3c625b5e8bf236b99.js?v=4cb3c625b5e8bf236b99:2)
    at async a.fetch (static/lab/jlab_core.4cb3c625b5e8bf236b99.js?v=4cb3c625b5e8bf236b99:2)
```

**What you expected to happen**:

Ideally no errors :)

**Minimal Complete Verifiable Example**:

```python
$ jupyter lab
```

and open the console.

**Anything else we need to know?**:

**Environment**:

This is with dask-labextension 5.0.1.

**Comment by ian-r-rose** [comment]
Thanks for the report @TomAugspurger! I can reproduce the error, though most of the extension still seems to function. To confirm -- does this break the extension entirely for you, or just some settings-related functionality?

This seems to not be a problem when installed with `pip`, so something is up with the conda packaging not bundling the settings schema.

**Comment by ian-r-rose** [comment]
This should be fixed in conda-forge/dask_labextension#22, thanks @bollwyvl ! I'll keep this open until I verify.

**Comment by ian-r-rose** [comment]
I've verified that this is fixed in 5.0.2 on `conda-forge`.
